src/server/routes.js

```diff
@@ -13,21 +16,80 @@ app.use(function (req, res, next) {
 app.use(express.json());
 app.use(express.urlencoded({ extended: true }));
 app.use(express.static(__dirname + "/../client/dist"));
+app.use(
+  cors({
+    origin: "*",
+  })
+);
```

Comment:
Were there still cors errors even with lines 9 --> 15? I find one or the other redundant unless they are both required.

Reply:
Removed the unnecessary line (19) app.use cors.

Updating the endpoints url name; will remove the unnecessary /delete part as the same route can be called with different methods in the front-end or with an ajax call. Since the route inventory/:pID/:sID can be called with different methods such as GET or DELETE, it is wiser to keep the routes as consistent as possible. 

Thanks the in depth review; I'll resolve this asap.

Reply:
Updated the endpoints to use /inventory/product. Additionally, I've adjusted the endpoints to handle the 400 error series. I.e A bad req.body will return 400 Bad Request, and exit immediately. I've also taken the liberty to add tests that will simulate these bad requests to see if it will return the proper 400 status.